core/src/main/java/com/netease/arctic/io/reader/CombinedDeleteFilter.java

```diff
@@ -240,6 +268,8 @@ private Predicate<StructForDelete<T>> applyEqDeletes() {
     return isInDeleteSet;
   }
 
+  abstract CloseableIterable<Record> readIdentifierData(Schema deleteSchema);
```

Comment:
DeleteFiter currently accepts positionPathSet as a parameter. The original idea behind this parameter was to avoid directly accepting DataFiles, but it seems a bit strange. So this time, RewriteInputs can be passed directly to DeleteFiter, eliminating the need for GenericCombinedIcebergDataReader to provide datafiles information. Hence, this abstract method is no longer necessary.

Reply:
Fixed.